Sources/StreamChatUI/ChatChannelList/ChatChannelListVC.swift

```diff
@@ -52,12 +61,20 @@ open class ChatChannelListVC: _ViewController,
     
     /// Reuse identifier of `collectionViewCell`
     open var collectionViewCellReuseIdentifier: String { String(describing: ChatChannelListCollectionViewCell.self) }
+    
+    /// Reuse identifier of `collectionViewCell`
+    open var collectionViewSkeletonCellReuseIdentifier: String { String(describing: ChatChannelListCollectionViewSkeletonCell.self) }
 
     /// Component responsible to process an array of `[ListChange<Item>]`'s and apply those changes to a view.
     private lazy var listChangeUpdater: ListChangeUpdater = CollectionViewListChangeUpdater(
         collectionView: collectionView
     )
     
+    /// Value of `channelListErrorView` height constraint.
+    var channelListErrorViewHeight: CGFloat { 88 }
```

Comment:
This number 88 seems pretty random too.

Reply:
That's the height from the design specs for iPhones from X onwards. Used a fixed height as a constraint when embedding the error view on the channel list, but I could also use a different constraint there. Currently working on the comments regarding the error view, but wanted to let you know where the number came from 👍🏻.

Reply:
If you tested on a device below iPhone X and it works good, then it's fine.

Reply:
Yes, it works fine on smaller devices and different orientations 👍🏻